integration-tests/typescript-koa/src/petstore-expanded.yml/generated.ts

```diff
@@ -67,8 +79,24 @@ export function bootstrap(
     }
 
     const { status, body } = await implementation.findPets(input, ctx)
+
+    ctx.body = responseValidationFactory(
```

Comment:
TODO: lift out of the request handler callback - we don't need to create these on every request

Reply:
Fixed by [eea4dd9](https://github.com/mnahkies/openapi-code-generator/pull/29/commits/eea4dd94c5412eaf9e7da27eff2f24678c367e5d) / [8aed680](https://github.com/mnahkies/openapi-code-generator/pull/29/commits/8aed680b5b4e9d7bb6b9448025662f5459ccb7ee)